src/withHelmet.js

```diff
@@ -0,0 +1,14 @@
+/* eslint react/display-name: 0 */
+import React from "react";
+import createElement from "recompose/createElement";
+import createHelper from "recompose/createHelper";
+import Helmet from "./Helmet.js";
+
+const withHelmet = (propsMapper, BaseComponent) => props => (
+  <div>
```

Comment:
Curious, what's the reason the Helmet component can't just `return this.props.children`? These empty divs tend to mess things up when `height: 100%` is desired further down.